services/crypto_update.proto

```diff
@@ -167,7 +167,7 @@ message CryptoUpdateTransactionBody {
     oneof virtual_address_update {
         /**
          * The virtual address to be added.
-         * The address one successfully addition becomes par tof the accounts ACTIVE virtual addresses
+         * The address one successfully addition becomes part of the accounts ACTIVE virtual addresses
```

Comment:
Once successfully added, the address becomes part of the account's ACTIVE virtual addresses.

Reply:
@kimbor fixed